interface/patient_file/encounter/forms.php

```diff
@@ -632,10 +636,10 @@ function divtoggle(spanid, divid) {
         echo "<div class='form_header_controls'>";
 
         // If the form is locked, it is no longer editable
-        if ( $esign->isLocked() ) {
-            echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-".attr($formdir)."-".attr($iter['id'])."'><span>".xlt('Locked')."</span></a>";
+        if ( $esign->isLocked() OR acl_check("groups","glog",false, 'write')) {
```

Comment:
fixed, 
'glog' is for group encounter view/write mode. 
this way we lock the user from viewing and using the  encounter log screen.